src/Microsoft.AspNetCore.Hosting.Abstractions/IWebHostBuilder.cs

```diff
@@ -33,6 +42,13 @@ public interface IWebHostBuilder
         IWebHostBuilder ConfigureServices(Action<IServiceCollection> configureServices);
```

Comment:
Yeah, I don't think we need to obsolete ConfigureServices, we aren't changing the IServiceCollection interface like ILoggerFactory. Most extension methods on IServiceCollection don't require anything from the Context today, so no reason to force everyone to accept the context that they don't need. You could make an argument for moving it to an extension method instead of being on the main WebHostBuilder though, if it isn't already.